backend/ondisconnect/src/app.ts

```diff
@@ -1,47 +1,12 @@
 import * as AWS from 'aws-sdk';
 import { APIGatewayProxyHandler } from 'aws-lambda/trigger/api-gateway-proxy';
+import { disconnect } from './disconnect';
 
 const ddb = new AWS.DynamoDB.DocumentClient({
   apiVersion: '2012-08-10',
   region: process.env.AWS_REGION,
 });
 
-const TABLE_NAME = process.env.TABLE_NAME || 'scrum_poker';
-
 export const handler: APIGatewayProxyHandler = async (event) => {
-  const deleteParams = {
-    TableName: TABLE_NAME,
-    Key: {
-      primaryKey: `connectionId:${event.requestContext.connectionId}`,
-    },
-  };
-  const queryItemParams = {
-    TableName: TABLE_NAME,
-    ConsistentRead: true,
-    Key: {
-      primaryKey: `connectionId:${event.requestContext.connectionId}`,
-    },
-  };
-
-  try {
-    const connectionItem = (await ddb.get(queryItemParams).promise()).Item;
-    await ddb.delete(deleteParams).promise();
-    if (connectionItem && connectionItem.groupId) {
-      const updateParams = {
-        TableName: TABLE_NAME,
-        Key: {
-          primaryKey: `groupId:${connectionItem.groupId}`,
-        },
-        UpdateExpression: 'REMOVE #1.connectionId',
-        ExpressionAttributeNames: {
-          '#1': connectionItem.userId,
-        },
-      };
-      await ddb.update(updateParams).promise();
-    }
-  } catch (err) {
-    return { statusCode: 500, body: 'Failed to disconnect: ' + JSON.stringify(err) };
-  }
-
-  return { statusCode: 200, body: 'Disconnected.' };
+  return await disconnect(ddb, event.requestContext.connectionId);
```

Comment:
I know there are different opinions on this for documentation purposes (is this async? But this is also implicitly encoded in the `APIGatewayProxyHandler` type), but just to note, `return await` is always equivalent to a plain `return`, and this whole function could be written as

```js
const handler = (event) => disconnect(ddb, event.requestContext.connectionId);
```